github-comment.yaml

```diff
@@ -1,18 +1,16 @@
 ---
 # https://github.com/suzuki-shunsuke/github-comment
-templates:
-  header: '{{env "TFACTION_TARGET"}} {{template "link" .}}'
```

Comment:
The function `env` was removed.

- https://github.com/suzuki-shunsuke/github-comment/releases/tag/v5.0.0
- https://github.com/suzuki-shunsuke/github-comment/pull/523